src/app/components/generic-error/generic-error.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode } from 'react';
+
+import { Header } from '@app/components/header';
+import { useRouteHeader } from '@app/common/hooks/use-route-header';
+
+import { GenericErrorLayout } from './generic-error.layout';
+
+interface ErrorProps {
+  body: string;
+  helpTextList: ReactNode[];
+  onClose?(): void;
+  title: string;
+}
+export function GenericError(props: ErrorProps) {
+  const { body, helpTextList, onClose, title } = props;
```

Comment:
Maybe setting default value for onClose here `onClose = () => window.close()` instead of the condition at line 23.

Reply:
Ok, yep, I can change it to a default value. 👍